compass_app/app/testing/fakes/repositories/fake_continent_repository.dart

```diff
@@ -1,14 +1,15 @@
 import 'package:compass_model/model.dart';
 import 'package:compass_app/data/repositories/continent/continent_repository.dart';
 import 'package:compass_app/utils/result.dart';
+import 'package:flutter/foundation.dart';
 
 class FakeContinentRepository implements ContinentRepository {
   @override
-  Future<Result<List<Continent>>> getContinents() async {
-    return Result.ok([
+  Future<Result<List<Continent>>> getContinents() {
+    return SynchronousFuture(Result.ok([
```

Comment:
Switched to using `SynchronousFuture` in tests when possible as I had issues with timeouts in widget tests